### PR TITLE
ES6 migration Arrow function added and fixed some unhandled rejection TypeError

### PR DESCRIPTION
**Pull request by ghost**
No new logic is added or rewritten any logic  
Only ES6 migration is done, arrow function is added. ref: #9589 
--update core/server/index.js
--core/index.js
--core/test/unit/lib/request_spec.js
updated change ref: https://github.com/TryGhost/Ghost/pull/9759/files/4c68d5124a2e7c1e79a220b499b0404761e023bb#r207944996

### CODE REVIEW

File: core/index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // ## Server Loader
 // Passes options through the boot process to get a server instance back
-var server = require('./server');
+let server = require('./server');
```
Comment (kirrg001) [inline core/index.js]: This is a const.

**Review by kirrg001 — CHANGES_REQUESTED** [review]
Thanks for your time! I left a comment.

File: core/index.js (hunk 1 above)
Comment (freezboltz) [reply core/index.js]: thanks for reviewing @kirrg001  change is in progress..

**Comment by freezboltz** [comment]
@kirrg001 please review line-40 core/server/update-check.js

**Comment by freezboltz** [comment]
What do you suggest for this? @kirrg001 waiting for your review...

File: core/index.js (hunk 1 above)
Comment (freezboltz) [reply core/index.js]: Already done by others.

**Review by ghost — COMMENTED** [review]
var-to-const 
core/index.js 
Already done

**Comment by kirrg001** [comment]
The PR is conflicting a lot with master. I'd suggest to re-create your PR branched of the current master state. Thanks for your help. I'm closing this PR now.

**Pull request closed without merging**